modules/serverless-kn-func-run.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * serverless/cli_tools/kn-func-ref.adoc
+// * serverless/functions/serverless-functions-getting-started.adoc
+
+:_content-type: REFERENCE
+[id="serverless-kn-func-run_{context}"]
+= Running a function locally
+
+You can use the `kn func run` command to run a function locally in the current directory or in the directory specified by the `--path` flag:
+
+.Example command to run a function in the current directory
+[source,terminal]
+----
+$ kn func run
+----
+
+.Example command to run a function in a directory specified as a path
+[source,terminal]
+----
+$ kn func run --path=<directory_path>
+----
+
+If the function that you are running has never previously been built, the `kn func run` command builds the function before running it by default.
```

Comment:
```suggestion
If the function that you are running has never previously been built, or if the project files have been modified since the last time it was built, the `kn func run` command builds the function before running it by default.
```